src/pages/networks/NetworkDetail.tsx

```diff
@@ -39,6 +37,21 @@ const NetworkDetail: FC = () => {
     return <Loader />;
   }
 
+  const tabs: string[] = ["Overview", "Configuration", "Forwards"].filter(
```

Comment:
Maybe something like this is a bit more clear?

```suggestion
  const getTabs = () => {
    if (!network?.managed) {
      return ["Overview"]
    }
    
    const type = network?.type ?? "";
    if (type === "physical") {
     return ["Overview", "Configuration"];
    }
    
    return ["Overview", "Configuration", "Forwards"];
  }
```